### PR TITLE
bug: HPA replicaCountProposal caculate error

### PR DESCRIPTION
**Pull request by volume-ji**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind bug

**What this PR does / why we need it**:
calculation error of HPA proposal pod replicas count.
**Which issue(s) this PR fixes**:

Nothing

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by volume-ji** [comment]
```release-note
none
```

**Comment by xichengliudui** [comment]
/kind bug
/sig testing
/priority backlog
release-note-none

File: pkg/controller/podautoscaler/horizontal_test.go [hunk 1]
```diff
@@ -1539,7 +1539,7 @@ func TestTolerancePerPodCMObject(t *testing.T) {
 		expectedConditions: statusOkWithOverrides(autoscalingv2.HorizontalPodAutoscalerCondition{
 			Type:   autoscalingv2.AbleToScale,
 			Status: v1.ConditionTrue,
-			Reason: "ReadyForNewScale",
```
Comment (Rajat-0) [inline pkg/controller/podautoscaler/horizontal_test.go]: why change this?

**Pull request closed without merging**